wgpu-core/src/track/mod.rs

```diff
@@ -315,6 +315,18 @@ impl<S: ResourceState> ResourceTracker<S> {
         }
     }
 
+    fn get<'a>(
```

Comment:
How do you feel about making `ref_count: Option<&RefCount>` argument in the existing `get_or_insert`, instead of adding a new method?